js_modules/dagit/packages/core/src/launchpad/LaunchpadSetupRoot.tsx

```diff
@@ -47,7 +47,7 @@ const LaunchpadSetupAllowedRoot: React.FC<Props> = (props) => {
   const queryString = qs.parse(window.location.search, {ignoreQueryPrefix: true});
 
   React.useEffect(() => {
-    if (queryString.config || queryString.mode || queryString.solidSelection) {
+    if (queryString.config || queryString.mode || queryString.solidSelection || queryString.tags) {
```

Comment:
Looks like `tags` could be an array, should this check its `length`?

Reply:
i think an empty array is fine